octobot_commons/asyncio_tools.py

```diff
@@ -69,6 +70,11 @@ def exception_handler(self, _, context) -> None:
         self.errors.append(context)
         if self.print_received_exceptions:
             print(context)
```

Comment:
Do we really need to use `print` ?

Reply:
to appear on pytest logs yes, this code is not used in the bot directly